src/app/utils.py

```diff
@@ -354,6 +354,21 @@ def createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseCommen
     return r.status_code
 
 
+def createErrorIssue(occured_at, made_by, error_message, license_text, token):
+    """ View for creating an GitHub issue
+    with the complete Error Report to the 
+    SPDX Online tools repository.
+    """
+  
+    body = "**1.** Occured At: {0}\n**2.** Made By: {1}\n**3.** Error Message: {2}\n**4.** License Text: {3}\n".format(occured_at, made_by, error_message, license_text)
+    title = "Error Report"
```

Comment:
Lets not directly assign a title. Keep a string constant. Similary for url - construct it from other url or keep it as setting in settings.py

Reply:
I kept the url as setting in the settings.py. What about the title of  the issue? Should we just kept it blank?

Reply:
Title should included license name/id